nomad/data_source_scheduler_config_test.go

```diff
@@ -13,28 +13,33 @@ func TestAccDataSourceSchedulerConfig_basic(t *testing.T) {
 		CheckDestroy: testFinalConfiguration,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccNomadSchedulerConfigSpread,
+				Config: testAccNomadDataSourceSchedulerConfigS,
 				Check: resource.ComposeTestCheckFunc(
```

Comment:
This test was broken and it seems to be flaky....I think it's clashing the the resource test since they are acting on the same cluster.

Reply:
Lets raise a follow up issue if there isn't one to track this.

Reply:
There's this issue: https://github.com/hashicorp/terraform-provider-nomad/issues/163 and previously failed attempt to fix them https://github.com/hashicorp/terraform-provider-nomad/pull/173 😬

But now I'm thinking of combining the two into a since test case. They are basically doing the same thing: test resource to write, test data source to read.

Reply:
Using a single test seems to have solved 🎉